### PR TITLE
Feature-98 UT test cases of History Manager

### PR DESCRIPTION
**Pull request by hiteshNEC**
View and merge UT test cases of HistoryManager.

### CODE REVIEW

File: History/HistoryManager/src/test/java/eu/neclab/ngsildbroker/historymanager/controller/HistoryControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+import eu.neclab.ngsildbroker.commons.enums.ErrorType;
+import eu.neclab.ngsildbroker.commons.exceptions.ResponseException;
+import eu.neclab.ngsildbroker.commons.ldcontext.ContextResolverBasic;
+import eu.neclab.ngsildbroker.commons.ngsiqueries.ParamsResolver;
+import eu.neclab.ngsildbroker.commons.stream.service.KafkaOps;
+import eu.neclab.ngsildbroker.historymanager.repository.HistoryDAO;
+import eu.neclab.ngsildbroker.historymanager.service.HistoryService;
+import eu.neclab.ngsildbroker.historymanager.utils.Validator;
+
+//@SpringBootTest(properties= {"spring.main.allow-bean-definition-overriding=true"})
```
Comment (amitrnec) [inline History/HistoryManager/src/test/java/eu/neclab/ngsildbroker/historymanager/controller/HistoryControllerTest.java:56]: Remove commented code

File: History/HistoryManager/src/test/java/eu/neclab/ngsildbroker/historymanager/controller/HistoryControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+					.accept(AppConstants.NGB_APPLICATION_JSONLD).content(temporalPayload))
+					.andExpect(status().isCreated());
+			verify(historyService, times(1)).createTemporalEntityFromBinding(any());
+		} catch (Exception e) {
+			Assert.fail(e.getMessage());
+		}
+	}
+
+	@Test
+	public void createTemporalEntityBadRequestTest() {
```
Comment (amitrnec) [inline History/HistoryManager/src/test/java/eu/neclab/ngsildbroker/historymanager/controller/HistoryControllerTest.java:139]: Please add doc comment in every method

File: History/HistoryManager/src/test/java/eu/neclab/ngsildbroker/historymanager/controller/HistoryControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+			Assert.fail(e.getMessage());
+		}
+	}
+	
+	/**
+	 * this method is use for update the temporalEntity
+	 */
+
+	@Test
+	public void updateAttrById() {
```
Comment (amitrnec) [inline History/HistoryManager/src/test/java/eu/neclab/ngsildbroker/historymanager/controller/HistoryControllerTest.java:173]: Add convenient name  same as other method

File: History/HistoryManager/src/test/java/eu/neclab/ngsildbroker/historymanager/controller/HistoryControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+							.content(temporalPayload))
+					.andExpect(status().isInternalServerError());
+		} catch (Exception e) {
+			Assert.fail(e.getMessage());
+		}
+	}
+	
+	@Test
+	public void modifyAttribInstanceTemporalEntityBadRequest() {
+
```
Comment (amitrnec) [inline History/HistoryManager/src/test/java/eu/neclab/ngsildbroker/historymanager/controller/HistoryControllerTest.java:243]: Please format the code and remove unused package if any

File: History/HistoryManager/src/test/java/eu/neclab/ngsildbroker/historymanager/service/HistoryServiceTest.java [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+			URI uri1=historyService.createTemporalEntityFromEntity(temporalPayload);
+			verify(kafkaOperations, times(3)).pushToKafka(any(),any(),any());
+		} catch (Exception e) {
+			Assert.fail();
+			e.printStackTrace();
+		}
+	}
+	
+	@Test
+	public void deleteTest() {
```
Comment (amitrnec) [inline History/HistoryManager/src/test/java/eu/neclab/ngsildbroker/historymanager/service/HistoryServiceTest.java:133]: Add convenient name  same as other method

**Review by amitrnec — APPROVED** [review]

**Review by ScorpioBroker — APPROVED** [review]
LGTM.

**Pull request merged**